app/oxide-rot-1/src/main.rs

```diff
@@ -17,6 +17,7 @@ fn main() -> ! {
     let (cycles_per_ms, _div) = get_clock_speed(&peripherals);
 
     startup(&core_peripherals, &peripherals);
+    lpc55_rot_startup::nuke_stack();
```

Comment:
I think this could be moved to just after `dice::run()` inside `startup()`.  That'd keep all the details about handing sensitive data in fewer crates.

Reply:
Alright. I've also moved the inline(never).